modules/arm_plugin/thirdparty/CMakeLists.txt

```diff
@@ -0,0 +1,170 @@
+# Copyright (C) 2020-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if(DEFINED ARM_COMPUTE_INCLUDE_DIR OR DEFINED ARM_COMPUTE_LIB_DIR)
+    if (NOT DEFINED ARM_COMPUTE_INCLUDE_DIR)
+        message(STATUS "Undefined ARM_COMPUTE_INCLUDE_DIR input variable should be set manually")
```

Comment:
> STATUS

`FATAL_ERROR`?

Reply:
@l-bat @apankratovantonp could you please comment this question?

Reply:
Yes FATAL_ERROR should be used